iset.mm

```diff
@@ -137143,6 +137318,15 @@ given prime (or other positive integer) that divides the number.  For
       XAUUTUSUUTYIYSUUEUVFUUKWJUUTYIYSUUFUVFUUQWJXOBWKWLVTUUTUURUUSYBUVCLYIXNWM
       YRXNXDWNWORWCWDWPWQ $.
 
+    $( Extended nonnegative integer closure of the general prime count
+       function.  (Contributed by Jim Kingdon, 13-Oct-2024.) $)
+    pcxnn0cl $p |- ( ( P e. Prime /\ N e. ZZ ) -> ( P pCnt N ) e. NN0* ) $=
```

Comment:
It would be nice to have this theorem in set.mm, too. Would it have any impact on ~pcgcd in set.mm?

Reply:
OK with me. Added.

As for impact, I didn't see easy opportunities to use it (there are cases which will naturally use ` RR* ` anyway, and places where ` RR* `  or ` ZZ u. { +oo } ` would be required rather than ` NN0* `). But I'll admit that I didn't look at everything.

Reply:
I also had a quick look, and did not find anything immediately. I just wanted to ask...